python/nano/test/run-nano-pytorch-inc-tests.sh

```diff
@@ -2,7 +2,7 @@
 
 export ANALYTICS_ZOO_ROOT=${ANALYTICS_ZOO_ROOT}
 export NANO_HOME=${ANALYTICS_ZOO_ROOT}/python/nano/src
-export INC_PYTORCH_NANO_TEST_DIR=${ANALYTICS_ZOO_ROOT}/python/nano/test/neural-compressor/pytorch
+export INC_PYTORCH_NANO_TEST_DIR=${ANALYTICS_ZOO_ROOT}/python/nano/test/neuralCompressor/pytorch
```

Comment:
why change the directory name?

Reply:
I think underscore is more appropriate

Reply:
But the name neural-compressor will conflicts with other modules